docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -168,6 +168,24 @@ In YAML:
       webservers:
         hosts:
           www[01:50].example.com:
+          
+You can specify a stride (increments between sequence numbers) when defining a numeric range of hosts:
+
+In INI:
+
+.. code-block:: text
+
+    [webservers]
+    www[01:50:2].example.com
```

Comment:
With this expression, does the webservers group here include ww01.example.com, ww03.example.com, ww05.example.com, and so on up to ww49.example.com? I'd add the "result" of the pattern to line 172, since it applies to both the INI and YAML formats.